### PR TITLE
Cleanup client comments

### PR DESCRIPTION
**Pull request by qdot**

### CODE REVIEW

File: Buttplug.Client/ButtplugWSClient.cs [hunk 1]
```diff
@@ -424,77 +490,87 @@ private void MessageReceivedHandler(object aSender, WebSocket4Net.MessageReceive
...
-        /// <summary>
-        /// Sends a message and returns the resulting message
-        /// </summary>
-        /// <param name="aMsg">Message to send.</param>
-        /// <returns>The response <see cref="ButtplugMessage"/></returns>
+        /// <summary>
+        /// Sends a message to the server, and handles asynchronously waiting for the reply from the server.
+        /// </summary>
+        /// <param name="aMsg">Message to send.</param>
+        /// <returns>The response, which will derive from <see cref="ButtplugMessage"/></returns>
```
Comment (blackspherefollower) [inline Buttplug.Client/ButtplugWSClient.cs]: Missing full stop

File: Buttplug.Client/ErrorEventArgs.cs [hunk 2]
```diff
@@ -5,40 +5,39 @@
...
 
-        /// <summary>
-        /// Initializes a new instance of the <see cref="ErrorEventArgs"/> class.
-        /// Based on an Error message
-        /// </summary>
-        /// <param name="aMsg">The Buttplug Error mesage</param>
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ErrorEventArgs"/> class, based on an Error message.
+        /// </summary>
+        /// <param name="aMsg">The Buttplug Error message</param>
```
Comment (blackspherefollower) [inline Buttplug.Client/ErrorEventArgs.cs]: Missing full stop

File: Buttplug.Core/Messages/Messages.cs [hunk 3]
```diff
@@ -177,7 +177,7 @@ public class DeviceMessageInfo
         /// </summary>
         [SuppressMessage("ReSharper", "MemberInitializerValueIgnored", Justification = "JSON.net doesn't use the constructor")]
         [JsonProperty(Required = Required.Always, NullValueHandling = NullValueHandling.Ignore)]
-        public Dictionary<string, MessageAttributes> DeviceMessages = new Dictionary<string, MessageAttributes>();
```
Comment (blackspherefollower) [inline Buttplug.Core/Messages/Messages.cs]: This is needed to stop JSON.net from setting this to null if empty (something to do with the reflection based construction). There's a suppression annotation above for it

File: docs/build.ps1 [hunk 4]
```diff
@@ -1,22 +1,31 @@
...
 }
 
 $docfxVersion = "2.33.0"
 & nuget install docfx.console -Version $docfxVersion -Source https://api.nuget.org/v3/index.json
 
 if($Deploy){
-	# Configuring git credentials
-	& git config --global credential.helper store
-	Add-Content "$env:USERPROFILE\.git-credentials" "https://$($env:git_access_token):x-oauth-basic@github.com`n"
+	  # Configuring git credentials
```
Comment (blackspherefollower) [inline docs/build.ps1]: I know that this is mostly from my frantic throwing this script together, but the whitespace seems off...

File: docs/build.ps1 [hunk 5]
```diff
@@ -1,22 +1,33 @@
+Write-Host "Branch: $env:APPVEYOR_REPO_BRANCH"
+Write-Host "PR Number: $env:APPVEYOR_PULL_REQUEST_NUMBER"
+Write-Host "Job Name: $env:APPVEYOR_JOB_NAME"
+if (-not (Test-Path env:git_access_token)) {
```
Comment (blackspherefollower) [inline docs/build.ps1]: why not just `if( $env:git_access_token )`

Comment (qdot) [reply docs/build.ps1]: Fixed.

**Pull request merged**